cosmic/resource_cosmic_egress_firewall_test.go

```diff
@@ -73,24 +73,24 @@ func TestAccCosmicEgressFirewall_update(t *testing.T) {
 						"cosmic_egress_firewall.foo", "rule.#", "2"),
 					resource.TestCheckResourceAttr(
 						"cosmic_egress_firewall.foo",
-						"rule.3593527682.cidr_list.1910468234",
+						"rule.316961573.cidr_list.2812511730",
 						CLOUDSTACK_NETWORK_1_IPADDRESS2+"/32"),
 					resource.TestCheckResourceAttr(
 						"cosmic_egress_firewall.foo",
-						"rule.3593527682.cidr_list.3378711023",
+						"rule.1790767400.cidr_list.522183319",
 						CLOUDSTACK_NETWORK_1_IPADDRESS1+"/32"),
 					resource.TestCheckResourceAttr(
-						"cosmic_egress_firewall.foo", "rule.3593527682.protocol", "tcp"),
+						"cosmic_egress_firewall.foo", "rule.316961573.protocol", "tcp"),
 					resource.TestCheckResourceAttr(
-						"cosmic_egress_firewall.foo", "rule.3593527682.ports.32925333", "8080"),
+						"cosmic_egress_firewall.foo", "rule.316961573.ports.32925333", "8080"),
 					resource.TestCheckResourceAttr(
 						"cosmic_egress_firewall.foo",
-						"rule.739924765.cidr_list.3378711023",
+						"rule.1790767400.cidr_list.522183319",
 						CLOUDSTACK_NETWORK_1_IPADDRESS1+"/32"),
 					resource.TestCheckResourceAttr(
-						"cosmic_egress_firewall.foo", "rule.739924765.protocol", "tcp"),
+						"cosmic_egress_firewall.foo", "rule.1790767400.protocol", "tcp"),
 					resource.TestCheckResourceAttr(
-						"cosmic_egress_firewall.foo", "rule.739924765.ports.1889509032", "80"),
+						"cosmic_egress_firewall.foo", "rule.1790767400.ports.1889509032", "80"),
 				),
 			},
 		},
```

Comment:
Can you undo the changes in this file? As you mentioned yourself, these are probably related to the input so for now I would like to leave them unchanged.